packages/components/src/components/input/input.component.ts

```diff
@@ -539,6 +570,40 @@ export default class SynInput extends SynergyElement implements SynergyFormContr
             <span part="suffix" class="input__suffix">
               <slot name="suffix"></slot>
             </span>
+
+            ${this.type === 'number' && !this.noSpinButtons
+              ? html`
+              <div part="stepper" class="input__number-stepper">
+                <button
+                  part="decrement-number-stepper"
+                  class="input__number-stepper-button"
+                  type="button"
+                  ?disabled=${isDecrementStepperDisabled || this.disabled}
+                  aria-hidden="true"
+                  ${longPress({ start: () => this.handleStepDown(), end: () => this.handleChange()})}
+                  tabindex="-1"
+                >
+                  <slot name="decrement-number-stepper">
```

Comment:
Good catch. Because of the naming collisions: I think we can name it the same. Because shoelace also does it (sometimes) see prefix and suffix naming of slot and part
But sometimes it takes different namings see clear-icon slot and clear-button css part

![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/94c40496-ea51-4f9e-8a76-660171f12c57)